eq-author/src/App/QuestionnairesPage/QuestionnairesView/Footer/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { colors } from "constants/theme";
+
+import PaginationNav from "./PaginationNav";
+
+const TableFooter = styled.div`
+  display: flex;
+  padding: 1em;
```

Comment:
The results text should be aligned with the table. You could drop the padding and use `margin: 1em 0;` to get the layout the same as the prototype.